physics/radiation_aerosols.f

```diff
@@ -4281,8 +4300,10 @@ subroutine aer_property_gocart                                    &
         enddo
         enddo
 
+        do k = 1, NLAY
```

Comment:
Nested do-loops also are easier to read with indentations between the loops

Reply:
Done as suggested.